vsintegration/src/vs/FsPkgs/FSharp.LanguageService/FSharp.LanguageService.Base/CodeWindowManager.cs

```diff
@@ -132,19 +124,15 @@ public class CodeWindowManager : IVsCodeWindowManager {
             }
         }
 #endif
-        /// <include file='doc\CodeWindowManager.uex' path='docs/doc[@for="CodeWindowManager.DropDownHelper"]/*' />
         /// <summary>Return the optional TypeAndMemberDropdownBars object for the drop down combos</summary>
         internal TypeAndMemberDropdownBars DropDownHelper {
             get { return this.dropDownHelper; }
         }
-        /// <include file='doc\CodeWindowManager.uex' path='docs/doc[@for="CodeWindowManager.CodeWindow"]/*' />
         /// <summary>Return the IVsCodeWindow associated with this code window manager.</summary>
         internal IVsCodeWindow CodeWindow {
             get { return this.codeWindow; }
         }
       
-        #region IVsCodeWindowManager methods
```

Comment:
for interface implementation region:

````csharp
#region IVsTextMarkerClient methods
public virtual void MarkerInvalidated() {
}

public virtual void OnBufferSave(string fileName) {
}

... lot of code, you scroll and dont know that interface is

#endregion
````

vs


````csharp
public virtual void IVsTextMarkerClient.MarkerInvalidated() {
}

public virtual void IVsTextMarkerClient.OnBufferSave(string fileName) {
}
````

on each method you know, useful when multiple interface or same name. And is F12 friendly too